docs/content/ingestion/batch-ingestion.md

```diff
@@ -137,6 +137,30 @@ s3n://billy-bucket/the/data/is/here/y=2012/m=06/d=01/H=01
 s3n://billy-bucket/the/data/is/here/y=2012/m=06/d=01/H=23
 ```
 
+
+##### `partition`
+
+A type of inputSpec that expects data from partitioned table of most SQL on Hadoops like Hive and Impala: `.../partition_col1=XXX/partition_col2=XXX/...`.
+
+|Field|Type|Description|Required|
+|-----|----|-----------|--------|
+|basePath|String|Base path of partitioned table|yes|
+|indexingPath|String|Base path to find partitioned data directories|no|
+|partitionColumns|List of String|List of Partition column names|no|
+
+If indexingPath is not specified, it has the same value as basePath.
+If partitionColumns are not specified, automatically ingest columns and values from directory path by finding `x=y` pattern and considering x as column name and y as column value.
+All partition columns are ingested as dimensions.
+
+If `basePath` is test, `indexingPath` is test/col1=a, `partitionColumns` are col1 and col2, and run with following input directories:
+```
```

Comment:
Perhaps one way to clean up the example is to have an example directory layout, and an example spec, and indicate what columns are included and ignored